resources/user-agents/browsers/phoenix/phoenix-generic.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
"Phoenix" is what Firefox/Firebird were originally called. It does have a limited version range as well, which we could add, but I felt the generic match would be enough.